### PR TITLE
Remove `dosth()` from readme examples

### LINKED ISSUE #133: remove `dosth()` from readme

### PR DESCRIPTION
**Pull request by UsingPython**
Close #133

### CODE REVIEW

**Comment by TimeForANinja** [comment]
Hey
sorry for the late reply
can you do me a favor and also replace
```js
const search = await ytsr('github', { pages: 1 });
display(search.items);
const r2 = ytsr.continueReq(search.continuation);
display(r2.items);
const r3 = ytsr.continueReq(r2.continuation);
display(r3.items);
```
with sth. like
```js
const firstResultBatch = await ytsr('github', { pages: 1 });
const secondResultBatch = ytsr.continueReq(firstResultBatch.continuation);
const thirdResultBatch = ytsr.continueReq(secondResultBatch.continuation);
// you can now use use the .items property off all Result Batches
```

**Comment by UsingPython** [comment]
I replaced it with your suggestion and added some `console.log`as example. I hope you agree with this.

File: README.md [hunk 1]
```diff
@@ -78,14 +76,14 @@ The previous request had to be done using `pages` limitation.
...
 ```js
-var ytsr = require('ytsr');
-
-const search = await ytsr('github', { pages: 1 });
-display(search.items);
-const r2 = ytsr.continueReq(search.continuation);
-display(r2.items);
-const r3 = ytsr.continueReq(r2.continuation);
-display(r3.items);
+const firstResultBatch = await ytsr('github', { pages: 1 });
```
Comment (TimeForANinja) [inline README.md:79]:
```suggestion
const ytsr = require('ytsr')
const firstResultBatch = await ytsr('github', { pages: 1 });
```

**Comment by UsingPython** [comment]
I will rebase and squash after approval.

File: README.md [hunk 2]
```diff
@@ -78,14 +76,16 @@ The previous request had to be done using `pages` limitation.
...
-display(r2.items);
-const r3 = ytsr.continueReq(r2.continuation);
-display(r3.items);
+const ytsr = require('ytsr');
+
+const firstResultBatch = await ytsr('github', { pages: 1 });
+const secondResultBatch = ytsr.continueReq(firstResultBatch.continuation);
+const thirdResultBatch = ytsr.continueReq(secondResultBatch.continuation);
+
+// You can now use the .items property off all result batches e.g.:
```
Comment (TimeForANinja) [inline README.md:85]:
```suggestion
// You can now use the .items property of all result batches e.g.:
```

**Comment by TimeForANinja** [comment]
other than that typo this looks fine to merge 👍

**Review by TimeForANinja — APPROVED** [review]

**Comment by TimeForANinja** [comment]
👍 thanks for the pr
might take a few more days though, until i can push to npm

**Pull request merged**